sigar_interface.go

```diff
@@ -353,19 +353,26 @@ type ProcIo struct {
 }
 
 type ProcMem struct {
-	Size        uint64
-	Resident    uint64
-	Share       uint64
-	MinorFaults uint64
-	MajorFaults uint64
-	PageFaults  uint64
+	Size          uint64
+	Resident      uint64
+	Share         uint64
+	MinorFaults   uint64
+	MajorFaults   uint64
+	PageFaults    uint64
+	PageFileBytes uint64
 }
 
 type ProcTime struct {
-	StartTime uint64
-	User      uint64
-	Sys       uint64
-	Total     uint64
+	CollectionTime time.Time
+	StartTime      uint64
+	User           uint64
+	Sys            uint64
+	Total          uint64
+
+	// Not valid until after Diff()
+	PercentUserTime  uint64
+	PercentSysTime   uint64
+	PercentTotalTime uint64
```

Comment:
nit: Maybe comment that these values are in the 0-100 range.  I guess that means they could also be 8-bit?  That's probably not a needed optimization